mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -248,12 +269,38 @@ class GBTClassificationModel private[ml](
     if (prediction > 0.0) 1.0 else 0.0
   }
 
+  override protected def predictRaw(features: Vector): Vector = {
```

Comment:
sorry, I'm still a little confused, should I override probability2prediction and simplify, or should I keep the argmax as is?  The argmax seems better because it is more general anyway, but please let me know if you would prefer that I make any changes here.